packages/frontend/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "lib": ["es2020", "DOM"],
```

Comment:
@sz-piotr I'm not quite sure why it worked before, but this has been `yarn typecheck` for me.

https://github.com/l2beat/dydx-state-explorer/runs/4726819953?check_suite_focus=true

![image](https://user-images.githubusercontent.com/15332326/148384591-eb3afe65-1916-452a-b82b-de6b90a0794c.png)